### PR TITLE
Added case studies section

### PR DESCRIPTION
**Pull request by MarsBarLee**
This is what I have so far. 
CaseStudies

The styling of the cards is based off [Joe's section2 tiles](https://github.com/numpy/numpy.org/pull/151). 

It's based on Shaloo's recent [mockup](https://github.com/numpy/numpy.org/issues/82#issuecomment-576116312)
image 

I have a few questions:

What section number should the case studies be? I currently have it as section3, so I have section3.html, section3.css and replaced section3 in the config.yaml file.

I also have pushed this branch on the numpy.org repo rather than my forked repo. Would it be better to make this pull request from my forked repo?

I've also added images to the content_images folder. I change them to placeholder images since I'm not sure if they're royalty free.

[Issue 129](https://github.com/numpy/numpy.org/issues/129)

### CODE REVIEW

**Comment by MarsBarLee** [comment]
There's a conflict as well. That must have been because my code was wrriten before Joe's section2 tiles were merged to the master. Should I update my local repo, make my changes and delete this pull request?

**Comment by joelachance** [comment]
@MarsBarLee I resolved the conflict, but there's a couple other things we may want to fix styling-wise. I'm a bit busy today, but will do a deeper dive this weekend.

**Comment by shaloo** [comment]
> This is what I have so far.
> CaseStudies
> 
> The styling of the cards is based off [Joe's section2 tiles](https://github.com/numpy/numpy.org/pull/151).
> 
> It's based on Shaloo's recent [mockup](https://github.com/numpy/numpy.org/issues/82#issuecomment-576116312)
> image
> 
> I have a few questions:
> 
> What section number should the case studies be? I currently have it as section3, so I have section3.html, section3.css and replaced section3 in the config.yaml file.
> 
> I also have pushed this branch on the numpy.org repo rather than my forked repo. Would it be better to make this pull request from my forked repo?
> 
> I've also added images to the content_images folder. I change them to placeholder images since I'm not sure if they're royalty free.
> 
> [Issue 129](https://github.com/numpy/numpy.org/issues/129)

@MarsBarLee, thanks for helping out with the tile page!  The mock up had links to these images - and source credits listed.  If that does not suffice and imposes royalty liabilities we can switch pictures to free ones available online from say unsplash...

Here are some examples:

Tesla: https://unsplash.com/photos/jwyO3NhPZKQ
Cricket ball : https://unsplash.com/photos/oDs_AxeR5g4

**Comment by shaloo** [comment]
> This is what I have so far.
> CaseStudies
> 
> The styling of the cards is based off [Joe's section2 tiles](https://github.com/numpy/numpy.org/pull/151).
> 
> It's based on Shaloo's recent [mockup](https://github.com/numpy/numpy.org/issues/82#issuecomment-576116312)
> image
> 
> I have a few questions:
> 
> What section number should the case studies be? I currently have it as section3, so I have section3.html, section3.css and replaced section3 in the config.yaml file.
> 
> I also have pushed this branch on the numpy.org repo rather than my forked repo. Would it be better to make this pull request from my forked repo?
> 
> I've also added images to the content_images folder. I change them to placeholder images since I'm not sure if they're royalty free.
> 
> [Issue 129](https://github.com/numpy/numpy.org/issues/129)

Regarding :

>>>

I also have pushed this branch on the numpy.org repo rather than my forked repo. Would it be better to make this pull request from my forked repo?

>>>

The process I've been using is to work with branches created out of my fork to keep updating changes until I am ready to issue a PR into NumPy.org - as suggested by @gommers. Once merge happens, I rebase my fork.  Jfyi, if that helps.

**Comment by rgommers** [comment]
Looks like a good start! I like the styling. The one issue I see when I build locally is that the tiles are very long vertically for some reason:

image

Regarding branches: let's keep the current one as is, but for future PRs it's indeed much better to do what @shaloo describes, pushing to your own fork and sending the PR from there.

**Comment by MarsBarLee** [comment]
Thanks for the comments! I'll update the images with the link @shaloo provided. 

It took a bit of research for me but I now understand how to pull and merge the changes that @joelachance made. 
I also realized I didn't squash my commits above- that's why there's 3 commit messages above.
I've also fixed the styling of the cards, as I didn't account for long screen sizes, as @rgommers has.

File: layouts/partials/css.html [hunk 1]
```diff
@@ -17,7 +17,16 @@
 <!-- Custom CSS -->
 <link rel="stylesheet" type="text/css" href="{{ "css/styles.css" | relURL }}" />
 <link rel="stylesheet" type="text/css" href="{{ "css/shell.css" | relURL }}" />
+<<<<<<< HEAD
```
Comment (rgommers) [inline layouts/partials/css.html]: This is an unresolved merge conflict. Normally git doesn't let you continue, somehow you managed to do that anyway @MarsBarLee. When this happens, you have to manually clean it up - deleting all `<<<<<`, `>>>>`, `=====` markers, and merging the content above and below `=====` so it makes sense again.

**Comment by rgommers** [comment]
Tiles look good to me now! Ready to go once the merge conflict left overs are cleaned up.

File: layouts/partials/css.html (hunk 1 above)
Comment (MarsBarLee) [reply layouts/partials/css.html]: Thanks for catching that! I cleaned up the merge conflicts in the other files but missed this one. I've resolved the merge conflict and committed it.

**Comment by rgommers** [comment]
This looks good design and code wise now. I had one minor comment left on naming: using `section3` everywhere isn't very descriptive, better to use something like `section-showcase` (or a shorter alternative). This can be done after merge though.

The other part I was trying to double check is image copyright (preferably we use fully unencumbered ones):
- black hole: that's the universal image so hard to replace, perhaps copyright EHT but we can do that on the case study page I think.
- gravitational waves: seems to be from https://sservi.nasa.gov/articles/scientists-detect-first-gravitational-waves/. Preferably we find a free one; otherwise it needs a credit like on that page (_Credit: R. Hurt/Caltech-JPL_)
- sports analytics: from Unsplash, which is fine
- self-driving car: copyright from http://www.hdcarwallpapers.com/tesla_roadster_4k-wallpapers.html, which is not great - @MarsBarLee I suggest to drop this tile for now, since we don't have content in the works yet and it's easy to add back later (with a copyright-free image)

File: static/css/section3.css [hunk 2]
```diff
@@ -15,10 +15,10 @@
 }
 
 .section3-box-content {
-    height: 450px;
-    width: 20vw;
+    height: 950px;
```
Comment (rgommers) [inline static/css/section3.css]: The changes in this file were unintended I think?

**Comment by shaloo** [comment]
@MarsBarLee, @rgommers : I was curious to see how the card tiles look so I checked out issues_129 branch.  However, I don't see tiles but a case-studies section after ecosystem tabs and 3 pictures one below other.  What am I missing?  Why don't I see three vertical tiles side by side?

Screenshot 2020-02-14 at 12 19 17 PM
Screenshot 2020-02-14 at 12 19 23 PM

File: static/css/section3.css (hunk 2 above)
Comment (rgommers) [reply static/css/section3.css]: I have pushed a commit to undo these changes.

**Comment by rgommers** [comment]
@shaloo those issues were introduced when fixing a merge conflict, I pushed a fix (@MarsBarLee please check!):

image

One issue left (other than adding summary text and linking the actual case studies), the 3 tiles aren't nicely centered below the section title.

**Comment by rgommers** [comment]
Added some text as well:

image

**Comment by InessaPawson** [comment]
> Added some text as well:
> 
> image
I love all the progress made in this section!

**Comment by MarsBarLee** [comment]
Hi all, I apologize for the delay. I've centered the columns and pulled the changes @rgommers has made. I learnt about rebasing commits, resolving merge conflicts and gitk. I was not able to squash my commits, as they were already pushed. Next time, I'll make to push fro my fork, pull changes from the upstream master, squash commits before pushing and rebase if necessary.

**Comment by rgommers** [comment]
Thanks Mars. No worries about git, you got it right now - that al sounds good:)

I'm not sure about the last commit, the tiles look a little uncomfortably spaced:

image

In `section2`, the horizontal space between tiles is only the `margin`, in `section3` it is `margin` plus all the space that's left over - the outer tiles are pushed out, while in `section2` they're pushed in. I don't quite see which CSS element(s) make the difference here, but could you try to make it behave the same as `section2`?

**Comment by MarsBarLee** [comment]
@rgommers Made those changes as you suggested.

**Comment by rgommers** [comment]
Looks good to me now, merged! Thanks @MarsBarLee

**Pull request merged**